cypress/support/pages/profile.pageObject.js

```diff
@@ -0,0 +1,14 @@
+import PageObject from '../PageObject';
+
+class ProfilePageObject extends PageObject {
+  get profilePageBio() {
+    return cy.getByDataCy('profile-bio');
```

Comment:
Missing a method to clear the database before each test. This will help ensure that each test is isolated and doesn't interfere with others.